src/BlazorAdmin/Pages/CatalogItemPage/List.razor.cs

```diff
@@ -23,8 +23,8 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
             if (firstRender)
             {
                 catalogItems = await new BlazorAdmin.Services.CatalogItemServices.ListPaged(Auth).HandleAsync(50);
-                catalogTypes = await new BlazorAdmin.Services.CatalogTypeServices.List(Auth).HandleAsync();
-                catalogBrands = await new BlazorAdmin.Services.CatalogBrandServices.List(Auth).HandleAsync();
+                catalogTypes = await new BlazorAdmin.Services.CatalogTypeServices.List(Auth, HttpClient).HandleAsync();
+                catalogBrands = await new BlazorAdmin.Services.CatalogBrandServices.List(Auth, HttpClient).HandleAsync();
```

Comment:
Why are we newing up these services here instead of injecting them?

Reply:
done